### ISSUE TITLE
System filenames should not be translated

### ISSUE THREAD

**Issue by marcbria** [pkp/pkp-lib#10123]
**Describe the bug**
The prefixes of the files in scheduledTaskLogs in the journals of my service depend on the language of the admin user... and it generates a babylon of files like the following: 

```
##admin.scheduledTask.depositDois##
##admin.scheduledTask.processQueueJobs##
##admin.scheduledTask.removeUnvalidatedExpiredUsers##
##admin.scheduledTask.reviewReminder##
##admin.scheduledTask.statisticsReport##
##admin.scheduledTask.subscriptionExpiryReminder##
##admin.scheduledTask.updateGeoDB##
AttivitàdelversanteinPKPPN
Avvisoperricordarelascadenzadell'abbonamento
CrossRefautomaticregistrationtask
CrossRef自动注册任务
DadosCrossRefdoremetente
DadosDataCitedoremetente
DataCiteautomaticregistrationtask
DataCite自动注册任务
DOAJautomaticregistrationtask
DOAJ自动注册任务
Latareanohageneradoningúnregistro.
LembretedeAvaliação
LembretedeExpiraçãodaAssinatura
##mailable.editorialReminder.description##
mEDRAautomaticregistrationtask
mEDRA自动注册任务
NomedeltaskcheregistrainautomaticoinCrossRef
NomedeltaskdiregistrazioneautomaticasuDataCite
NomedeltaskdiregistrazioneautomaticasuDOAJ
Nomedeltaskperilcaricodellestatistiched'uso
Notificacióndeaccesoabierto
OpenAccessNotification
PKPPNDepositorTask
PLNdoPKP
##plugins.generic.pln.depositorTask.name##
##plugins.generic.usageStats.usageStatsLoaderName##
##plugins.importexport.datacite.senderTask.name##
##plugins.importexport.doaj.senderTask.name##
##plugins.importexport.medra.senderTask.name##
Publicarlestramesesprogramadesperapublicació
Publicarlosenvíosprogramadosparasupublicación
Publishsubmissionsscheduledforpublication
Rappeld'évaluation
Rappeld'expirationd'abonnement
Recordatoridecaducitatdelasubscripció
Recordatoriderevisió
Recordatoriodecaducidaddelasuscripción
Recordatorioderevisión
RegistrazioneautomaticaDOImEDRA
Removemucholderfailedjobsfromfailedjoblist.
Reviewreminder
ReviewReminder
Sollecitoperilrevisore
SubscriptionExpiryReminder
Tâcheautomatiqued'enregistrementCrossRef
Tâcheautomatiqued'enregistrementDataCite
Tâcheautomatiqued'enregistrementdeCrossRef
Tâchedechargementdufichierdestatistiquesd'utilisation
TâchededépôtPKPPLN
Tâched'enregistrementautomatiqueDOAJ
Tâched'enregistrementautomatiquemEDRA
Tâchedeversementdufichierdestatistiquesd'utilisation
TareaautomáticaderegistroenCrossRef
Tareadecargadearchivodeestadísticasdeuso
TareadedepósitodePKPPN
TareaderegistroautomáticoDataCite
TareaderegistroautomáticodeDOAJ
TareaderegistroautomáticodemEDRA
TarefaderegistroautomáticodoDOAJ
TarefaderegistroautomáticodomEDRA
Tarefadocarregadordearquivodeestatísticasdeuso
Tascadecàrregad'arxiusd'estadístiquesd'ús
TascadedipòsitdePKPPN
TascaderegistreautomàticDataCite
TascaderegistreautomàticdeCrossRef
TascaderegistreautomàticdeDOAJ
TascaderegistreautomàticdemEDRA
Usagestatisticsfileloadertask
订阅到期提醒
评审提醒
```

Some issues could be noticed here:
- Inconsistent log filenames. The system admin would like tot select the lang independently of the admin user election.
- Prefixes not translated: as admin select a lang that don't include the prefix.
- Filenames with accents, apostrophes and special chars.
- Filenames in langs what the system admin don't know.

**What application are you using?**
OJS 3.2 and 3.3.

**Additional information**
This is kind of related with the issues that talk about filename of other log files and the downloaded files (in the workflow or as a visitor) that are also translated... generating duplicates in files with different names and issues when prefixes are not translated.

In general, my suggestion would be make it easier and don't make filenames is dependent of the user preferences.
A global OJS config var to define a lang for the whole system would be better.

**Comment by jonasraoni** [comment]
I'll close this as a duplicate of https://github.com/pkp/pkp-lib/issues/9979, which should be present on the next releases (the next release will probably happen within this month).

**Comment by marcbria** [comment]
Great. It had been in my top 10 uncomfortable things for years... and I could never find the time to post it and today that I finally decided to publish it, it was posted by a fellow one month ago. :-)

Great news this is finally fixed. Thanks Jonas.

Just to be sure: 
- [Considered a bug, so also fixed in 3.3](https://github.com/pkp/pkp-lib/issues/9979#issuecomment-2161428965), yes?
- [Filenames will be set english only](https://github.com/pkp/pkp-lib/issues/9979#issuecomment-2144844588), yes?

If the answer is yes to both, you made me so happy. :-)

**Comment by jonasraoni** [comment]
Yep, this has been fixed in `stable-3_3_0`, and it will use the class name instead of translated content, I also think that the content of those files should have messages in English, to also avoid the `##something.failed##`, I'll create another issue for that.
